frontend/scss/app/components/_icon_button.scss

```
@@ -24,6 +26,36 @@
   }
 
+  @media (prefers-reduced-motion: no-preference) {
+    transition: transform 120ms ease;
+
+    &::after {
+      content: "";
+      position: absolute;
+      inset: 0;
+      pointer-events: none;
+      border-radius: inherit;
+      background-image: radial-gradient(circle, currentColor 12%, transparent 12.5%);
```

Stylelint (17.12.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)